js/packages/glimmer-idc/src/components/collections.ts

```diff
@@ -0,0 +1,45 @@
+import Component, { hbs, tracked } from '@glimmerx/component';
+import TitleBar from './title-bar';
+import List from './list';
+import ListItem from './list-item';
+import ListSpinner from './list-spinner';
+
+interface Args {}
+
+export default class Collections extends Component<Args> {
+  @tracked title: string = 'All Collections';
+  @tracked isLoading: boolean = false;
+  @tracked list: {}[] = [];
+
+  constructor(owner: unknown, args: Args) {
+    super(...arguments);
+
+    this.fetchCollections();
+  }
+
+  async fetchCollections() {
+    this.isLoading = true;
+    let url =
+      'https://islandora-idc.traefik.me/search_rest_endpoint?query=ss_type:collection_object';
```

Comment:
I think this should be a relative URL or have the base URL configurable. Something like `let url = '/search_rest_endpoint?query=...'` might be OKish. We might want to think about seeing if Drupal can give us the rest endpoints through its API or something.

Hard coding the URL should be fine now, as it's early on.

Reply:
Yeah makes good sense. Was hoping to update this when we were working on the search related aspects.